content/api/rest-api.textile

```diff
@@ -1152,6 +1153,56 @@ A successful request returns an empty response.
 
 An unsuccessful request returns an error.
 
+h3(#push-publish-batch). Publish via batch push API
+
+Convenience endpoint to deliver multiple push notification payloads to multiple devices or browsers in a single request by specifying a list of recipients and corresponding payloads.
+Currently the batch push endpoint allows a maximum of 10,000 recipients per request (recipients are counted per payload, so publishing two payloads to the same recipient counts as two recipients).
```

Comment:
Should we change this wording from "recipients" to "notifications" as the thing which is limited?
E.g.: "Currently the batch push endpoint allows a maximum of 10,000 notifications per request (one notification is counted as one payload delivered to one recipient, so publishing two payloads to the same recipient counts as two notifications)."